client/extensions/woocommerce/woocommerce-services/lib/pdf-label-utils/index.js

```diff
@@ -61,5 +61,5 @@ export const getPrintURL = ( paperSize, labels ) => {
 };
 
 export const getPreviewURL = ( paperSize, labels ) => {
-	return _getPDFURL( paperSize, labels, api.url.labelTestPrint() );
+	return _getPDFURL( paperSize, labels, api.url.labelsTestPrint() );
```

Comment:
This was broken, there is no `labelTestPrint` (single `label`) exported in https://github.com/Automattic/wp-calypso/blob/trunk/client/extensions/woocommerce/woocommerce-services/api/url.js. Webpack now detects missing second-level exports.